templates/home/index.html

```diff
@@ -417,7 +417,7 @@ <h1 data-aos="fade-up">Contact Us</h1>
         </div>
     </section>
     <div class="con-sidenav">
-        <a href="#contact-us">Contact Us</a>
+        <a href="#contact-us"><i class="fas fa-phone-square"></i>Contact Us</a>
```

Comment:
Instead of `fa-phone-square`, I think `fa-phone` will look better.

Reply:
I have changed the icon and angle it in commit 32f4df07d4fae1fb65a1c085b2c77951da84ce6a. Please review